UnitTests/Suggestions/ViewModel/SuggestionContainerViewModelTests.swift

```diff
@@ -23,8 +23,30 @@ import Suggestions
 
 final class SuggestionContainerViewModelTests: XCTestCase {
 
+    let suggestionLoadingMock = SuggestionLoadingMock()
```

Comment:
Probably better to initialise this in the setUp method? We should also nil everything in tearDown. Reason is that XCTestCase subclass gets instantiated for every test and gets deallocated only when all test in the entire app are finished running. So memory balloons and CI could become slower.